ui/client/components/graph/node-modal/ExpressionSuggest.js

```diff
@@ -8,10 +8,9 @@ import ProcessUtils from '../../../common/ProcessUtils';
 import ExpressionSuggester from './ExpressionSuggester'
```

Comment:
Please remove all semicolons.

Reply:
done - but removed only in imports, I don't want to touch whole file because I want to have option to quick revert this change in case of some problems with new completion